### ISSUE TITLE
Use user token to login?

### ISSUE THREAD

**Issue by AJolly** [dhrumilp15/haystackfs#136]
Is it possible to use a normal user account for this?
Normal discord search tries to be "smart" and does not let you do exact string searches. 
For example, searching for "liquidate" pulls up "liquidity". 

This causes me all sorts of problems.

**Comment by dhrumilp15** [comment]
Definitely, pushing this to a normal user account is the most ideal implementation of this project I think. I do mean to actually devote a good amount of time to building this, and I'll update the [discord server](https://discord.gg/b9Vvn4hZrD) when I do!
